modis_tools/resources.py

```diff
@@ -24,19 +24,8 @@ def __init__(
     def query(self, **kwargs) -> List[Collection]:
         resp = self.no_auth.get(params=kwargs)
         try:
-            iter_dict = resp.json()["feed"].copy()
-            sanitize_inds = []
-            for i, entry in enumerate(iter_dict["entry"]):
-                for j, link in enumerate(entry["links"]):
-                    if " " in link["href"]:
-                        sanitize_inds.append((i, j))
-            if len(sanitize_inds) > 0:
-                for i, j in sanitize_inds:
-                    link_to_clean = iter_dict["entry"][i]["links"][j]["href"]
-                    iter_dict["entry"][i]["links"][j]["href"] = link_to_clean.replace(
-                        " ", "%20"
-                    )
-            collection_feed = CollectionFeed(**iter_dict)
+            feed = resp.json()["feed"]
+            collection_feed = CollectionFeed(**feed)
```

Comment:
this was originally:

```suggestion
            collection_feed = CollectionFeed(**resp.json()["feed"])
```